src/app/content/components/BookBanner.tsx

```diff
@@ -57,6 +57,11 @@ export interface PropTypes {
   hasUnsavedHighlight?: boolean;
 }
 
+interface BookBannerState {
```

Comment:
This interface could be placed directly above BookBanner class

Reply:
I moved the props too since they are far away from it as well